datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -151,6 +160,7 @@ func (f *ChainDataFetcher) startFetching() error {
 	if f.fetchingStarted {
 		return errors.New("fetching is already started")
 	}
+	f.fetchingStarted = true
```

Comment:
It would be better if there are comments that [1] line 163~170 is to fetch the latest blocks and [2] line 171~174 is to fetch the blocks between the checkpoint and the current head block.

Reply:
Added comments. 24c0b1a8a81e0233f22758be6f8f7f6e5264c723